runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -79,7 +78,8 @@ def __init__(
             dryrun=dryrun,
             **kwargs,
         )
-
+        self._creds = kwargs.get("creds", None)
+        self.address = kwargs.get("head_ip", None)
```

Comment:
address is a property that gets set in the parent Cluster class, how come we're overriding that here?